### ISSUE TITLE
[fix] npm i @koa/send 

### ISSUE THREAD

**Issue by vanodevium** [koajs/send#149]
When I enter:

```sh
npm i @koa/send
```

I see:
```sh
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@koa%2fsend - Not found
npm ERR! 404 
npm ERR! 404  '@koa/send@*' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

## Checklist

- [x] I have searched through GitHub issues for similar issues.
- [x] I have completely read through the README and documentation.

**Comment by vanodevium** [comment]
Thanx!

**Comment by masterflitzer** [comment]
the readme should document `koa-send` or the package should be published under `@koa/send`, currently this is very confusing

**Comment by mreinstein** [comment]
I agree, and the solution I'd like to see is the npm module getting published but koajs and express are both mausoleums and effectively dead/dying so I'm not super hopeful these will happen any time soon.

**Comment by mreinstein** [comment]
Still waiting on https://github.com/koajs/koa-range to get published too.

**Comment by masterflitzer** [comment]
> I agree, and the solution I'd like to see is the npm module getting published but koajs and express are both mausoleums and effectively dead/dying so I'm not super hopeful these will happen any time soon.

dying? what is used as alternative then? i just migrated 2 small projects to koa cause i thought it might be leaner than express

**Comment by vanodevium** [comment]
Please investiage [ranked list](https://github.com/vanodevium/node-framework-stars)
---
I still use koajs, nest.js uses koajs. It's important.

But, as alternative, you can use fastify at any time.

**Comment by masterflitzer** [comment]
thx for the tip

**Comment by mreinstein** [comment]
> i just migrated 2 small projects to koa cause i thought it might be leaner than express

it's definitely leaner than express, and both projects are still quite usable for now.

**Comment by 3imed-jaberi** [comment]
[@koa/send published with v6.0.0 few minutes ago](https://www.npmjs.com/package/@koa/send).
> koa-send wait for permissions and it should be soon published with v6.

cc @titanism
